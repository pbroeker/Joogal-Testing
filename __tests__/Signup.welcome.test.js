import React from 'react';
import Welcome from '../Signup/Signup.Welcome';
import { render } from '@testing-library/react-native';

describe('Signup Welcome', () => {
  const setSignup = jest.fn();
  const navigation = {navigate : setSignup};

  test('snapshot', () => {
    const { toJSON } = render( <Welcome navigation={navigation} /> );
    expect(toJSON()).toMatchSnapshot();
  })
})