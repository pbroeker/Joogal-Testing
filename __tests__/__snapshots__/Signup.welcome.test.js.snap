// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup Welcome snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#1d1e20",
      "height": "100%",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "marginTop": "20%",
        "position": "absolute",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "whitesmoke",
          "fontFamily": "PoiretOne_400Regular",
          "fontSize": 30,
          "marginBottom": 30,
          "marginLeft": "10%",
          "width": "52%",
        }
      }
    >
      Welcome to Joogal.
    </Text>
    <Text
      style={
        Object {
          "color": "whitesmoke",
          "fontFamily": "PoiretOne_400Regular",
          "fontSize": 30,
          "marginBottom": 30,
          "marginLeft": "10%",
          "width": "52%",
        }
      }
    >
      Let's find your next collaboration!
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 370,
          "marginTop": 261,
          "width": 375,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "marginTop": 392,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "teal",
          "opacity": 1,
          "padding": 15,
          "width": "100%",
        }
      }
      testID="welcomeContinueButton"
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 20,
            "fontWeight": "600",
          }
        }
      >
        Continue
      </Text>
    </View>
  </View>
</View>
`;
